frontend/src/components/samples/SamplesDetailContent.js

```diff
@@ -78,11 +79,11 @@ const SamplesDetailContent = ({samplesByID, sampleKindsByID, containersByID, pro
     listVersions(sample.id);
 
   if (isLoaded && !isProcessesEmpty && !sample.isFetching) {
-    sample.processes_samples.forEach((id, i) => {
+    sampleProcessSamples.forEach((id, i) => {
       withProcess(processesByID, id, process => process.id);
     })
-    sample.processes_samples.forEach((id, i) => {
-      sampleProcesses.push(processesByID[id])
+    sampleProcessSamples.forEach((id, i) => {
+      processSamples.push(processesByID[id])
```

Comment:
Loop once and do both tasks.

Can this put `undefined`s in `processSamples` if the process data hasn't been completed yet?